lib/ios_analytics/translate.rb

```diff
@@ -3,9 +3,36 @@
 
 module IosAnalytics
   module Translate
-    def trans(checkstyle_dom, plist_file)
+    def trans(derived_data, app_name)
+      doc = REXML::Document.new
+      doc << REXML::XMLDecl.new('1.0', 'UTF-8')
+
+      checkstyle = doc.add_element('checkstyle')
+
+      path = Pathname.new(derived_data)
+        .join(
```

Comment:
__[rubocop]__
* Align the operands of an expression in an assignment spanning multiple lines.